src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -698,10 +698,8 @@ namespace DataHeatBalance {
         int zoneOABalanceIndex = 0;       // Index to ZoneAirBalance for this zone, if any
 
         // Spaces
-        bool anySurfacesWithoutSpace = false; // True if any surfaces in a zone do not have a space assigned in input
-        bool anySurfacesWithSpace = false;    // True if any surfaces in a zone have a space assigned in input
```

Comment:
Unrelated to the main defect, but moved these out of state.

Reply:
:+1: